lib/states/tx/houston.R

```diff
@@ -3,82 +3,147 @@ source("common.R")
 
 # VALIDATION: [YELLOW] The Houston PD's Annual Reports don't list traffic
 # figures, but the stop counts don't appear unreasonable for a city of 2M
-# people. 2018 only has partial data.
+# people. 2018 only has partial data. 
+# NEW DATA UPDATE: Number of stops on 2019 is roughly on par with number
+# of stops in 2017. New data for 2018 is the same as old 2018 data (need to fix). 
 load_raw <- function(raw_data_dir, n_max) {
-  d <- load_years(raw_data_dir, n_max = n_max)
-  bundle_raw(d$data, d$loading_problems)
+  
+  # old data 
+  old_2014 <- load_single_file(raw_data_dir, '2014.csv', n_max = n_max)
+  old_2015 <- load_single_file(raw_data_dir, '2015.csv', n_max = n_max)
+  old_2016 <- load_single_file(raw_data_dir, '2016.csv', n_max = n_max)
+  old_2017 <- load_single_file(raw_data_dir, '2017.csv', n_max = n_max)
+  
+  # new data: note that old data only had part of 2018, so we use new 
+  # data for 2018, which has the full year
+  new_2018 <- load_single_file(raw_data_dir, '2018.csv', n_max = n_max) 
+  new_2019 <- load_single_file(raw_data_dir, '2019.csv', n_max = n_max)
+  new_2020 <- load_single_file(raw_data_dir, '2020.csv', n_max = n_max)
+  
+  old_d <- bind_rows(
+    old_2014$data,
+    old_2015$data,
+    old_2016$data,
+    old_2017$data
+  )
+  
+  new_d <- bind_rows(
+    new_2018$data,
+    new_2019$data,
+    new_2020$data %>% rename("DISPOTION" = "DISPOSITION_NAME")
```

Comment:
Having nested pipes gets a little confusing to me, although there are times when it can make the code more readable. (See: https://style.tidyverse.org/pipes.html#short-pipes.) In this case, the way that Dan and Amy dealt with this sort of issue most often was to simply to avoid renaming the columns in `load_raw` and then `mutate(DISPOSITION = coalesce(DISPOSITION, DISPOSITION_NAME))` in `clean`.